releases.md

```diff
@@ -16,39 +16,30 @@ _Updated April 26, 2023_
 
 For more information on the changes planned for each release, please see the [Project Roadmap](https://github.com/orgs/opensearch-project/projects/1)
 
-### 2023 ###
 
 Note:  We have not added a major release to the 2023 schedule yet.  If/when we add one, it will replace a minor release in the 2.x line.  See below for criteria for a major releases.
 
+
 <div class="table-styler"></div>
 
-| Release Number| Code Freeze Date | Release Date   |
-|:--------------|:-----------------|:---------------|
-| 2.5.0         | January 10th     | January 24th   |
-| 1.3.8         | January 26th     | February 2nd   |
-| 2.6.0         | February 21th    | February 28th  |
-| 1.3.9         | March 9th        | March 16th     |
-| 2.7.0         | April 17th       | May 2nd        |
-| 1.3.10        | May 11th         | May 18th       |
-| 2.8.0         | May 30th         | June 6th       |
-| 1.3.11        | June 22nd        | June 29th      |
-| 2.9.0         | July 11th        | July 24th      |
-| 1.3.12        | August 3rd       | August 10th    |
-| 2.10.0        | September 05th   | TBD            |
```

Comment:
I don't think we want to delete this history as we move it to a 'Release History' section at the end of the year.

Reply:
I thought about it, and decided that most folks didn't care, and if they did, they could pull the history off the page.   We don't, for example, show all the earlier release dates like 1.0-1.3.   If you think folks would like a release history page, do you want to make one?